.github/workflows/publish_pypi.yml

```diff
@@ -212,6 +213,35 @@ jobs:
           path: ./dist/*.tar.gz
           if-no-files-found: error
 
+  publish_anaconda:
+    # This job is only of value to PyBaMM and would always be skipped in forks
+    # It is meant to run on a schedule, when manually triggered, and does not
+    # run on releases.
```

Comment:
Note: this job is configured differently and is isolated from the PyPI upload job for the reason that the latter uses the `pypi` repository environment, which must not be accessed by other jobs.